src/config/dynamic-theme-fixes.config

```diff
@@ -901,6 +901,11 @@ drive.google.com
 INVERT
 img[src$="google_gsuite"]
 
+CSS
+.ndfHFb-c4YZDc.ndfHFb-c4YZDc-i5oIFb .ndfHFb-c4YZDc-Bz112c:not([onclick]):not(:link):not(:visited) {
+ background-image: url('//ssl.gstatic.com/docs/common/viewer/v3/v-sprite26.svg')!important;
```

Comment:
4 Spaces
Space between `)` and `!important`

Reply:
Done.